lib/astarte_core/triggers/simple_trigger_config.ex

```diff
@@ -261,6 +275,72 @@ defmodule Astarte.Core.Triggers.SimpleTriggerConfig do
     end
   end
 
+  defp validate_introspection_triggers_interface_version(%Ecto.Changeset{} = changeset) do
+    if allows_interface_match?(changeset) do
+      validate_interface_name_and_major(changeset)
+    else
+      changeset
+    end
+  end
+
+  defp validate_interface_name_and_major(%Ecto.Changeset{} = changeset) do
```

Comment:
Call this `validate_introspection_trigger_interface` since there's already a `validate_interface` and as it is right now it's not clear what's the difference between the two